packages/headless/package.json

```diff
@@ -30,14 +30,15 @@
     "doc:parse": "ts-node --project ./doc-parser/tsconfig.build.json ./doc-parser/doc-parser.ts"
   },
   "dependencies": {
-    "@coveo/bueno": "^0.14.0",
+    "@coveo/bueno": "^0.13.0",
     "@reduxjs/toolkit": "^1.5.0",
     "@types/pino": "^6.3.4",
     "@types/redux-mock-store": "^1.0.2",
     "coveo.analytics": "^2.18.19",
     "cross-fetch": "^3.0.6",
     "dayjs": "^1.9.6",
     "exponential-backoff": "^3.1.0",
+    "fast-equals": "^2.0.3",
```

Comment:
https://coveo.slack.com/archives/G7HD9P48H/p1621604066018200

This could be used, in some instance, for the subscribe method, like in this PR.